File: CPP/Program/Baju.cpp
#include "Aksesoris.cpp"

class Baju : public Aksesoris {
private:
    string untuk;
    string size;
    string merk;

public:
    // constructors
    Baju() : Aksesoris(), untuk(""), size(""), merk("") {} // default constructor
    Baju(int id, string nama_produk, double harga_produk, int stok_produk, string jenis, string bahan, string warna, string untuk, string size, string merk)
        : Aksesoris(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna), untuk(untuk), size(size), merk(merk) {}

    // getters
    string getUntuk() { return untuk; }
    string getSize() { return size; }
    string getMerk() { return merk; }

    // setters
    void setUntuk(string untuk) { this->untuk = untuk; }
    void setSize(string size) { this->size = size; }
    void setMerk(string merk) { this->merk = merk; }
};

File: PHP/Program/Baju.php
<?php

require_once ('Aksesoris.php');

class Baju extends Aksesoris {
    private $untuk;
    private $size;
    private $merk;

    // default constructor
    public function __construct($id = 0, $nama_produk = "", $harga_produk = 0.0, $stok_produk = 0, $jenis = "", $bahan = "", $warna = "", $untuk = "", $size = "", $merk = "", $foto_produk = "") {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $jenis, $bahan, $warna, $foto_produk);
        $this->untuk = $untuk;
        $this->size  = $size;
        $this->merk  = $merk;
    }

    // getters
    public function getUntuk() {
        return $this->untuk;
    }

    public function getSize() {
        return $this->size;
    }

    public function getMerk() {
        return $this->merk;
    }

    // setters
    public function setUntuk($untuk) {
        $this->untuk = $untuk;
    }

    public function setSize($size) {
        $this->size = $size;
    }

    public function setMerk($merk) {
        $this->merk = $merk;
    }
}
